index.md

```diff
@@ -23,6 +23,21 @@ Join #cri-o on IRC (freenode)
 * **RHEL7**: Soon
 * **Centos**: Soon
 
+#### System Container
+
+* **Fedora and Fedora Atomic Host**: Available on 26+ and Rawhide
+
+        atomic install --system-package=no -n cri-o --storage ostree
+docker.io/gscrivano/cri-o-fedora
+
+* **Centos and Centos Atomic Host**: Available 7.4+
+
+        atomic install --system-package=no -n cri-o --storage ostree
+docker.io/gscrivano/cri-o-centos
```

Comment:
we could probably already use the CentOS registry location here.

We can update the Fedora registry once we have it.

Reply:
@giuseppe Can you paste the CentOS location here?

Reply:
here it is: registry.centos.org/projectatomic/cri-o:latest

Reply:
@jtligon 

```
atomic install --system-package=no -n cri-o --storage ostree registry.centos.org/projectatomic/cri-o:latest
```